forms/gridfield/GridFieldExportButton.php

```diff
@@ -111,7 +111,11 @@ public function generateExportFileData($gridField) {
 			// determine the CSV headers. If a field is callable (e.g. anonymous function) then use the
 			// source name as the header instead
 			foreach($csvColumns as $columnSource => $columnHeader) {
-				$headers[] = (!is_string($columnHeader) && is_callable($columnHeader)) ? $columnSource : $columnHeader;
+				if (is_array($columnHeader) && array_key_exists('title', $columnHeader)) { 
+					$headers[] = $columnHeader['title'];
+				 } else { 
+					$headers[] = (!is_string($columnHeader) && is_callable($columnHeader)) ? $columnSource : $columnHeader; 
```

Comment:
I don't think it is. `is_callable` will also return `true` if the name is an existing function. I think that's why the original author of this code used `is_string` as well.